CHANGELOG.md

```diff
@@ -89,6 +89,7 @@ This release containts the following new modules:
   * [#4388](https://github.com/osmosis-labs/osmosis/pull/4388) Increase the max allowed contract size for non-proposal contracts to 3MB
   * [#4384](https://github.com/osmosis-labs/osmosis/pull/4384) migrate stXXX/XXX constant product pools 833, 817, 810 to stable swap
   * [#4461](https://github.com/osmosis-labs/osmosis/pull/4461) added rate limit quotas for a set of high value tokens
+  * [#4832](https://github.com/osmosis-labs/osmosis/pull/4832) Add CL simulator test
```

Comment:
I don't think we need a changelog entry for this. This is dev-focused, the end users should not be concerned about this change

```suggestion
```